README.md

```diff
@@ -47,46 +29,9 @@ Additional info on Azure deployment models [https://azure.microsoft.com/en-us/do
 
```

Comment:
As per our gemspec, azure supports 1.9.3 and above. `s.required_ruby_version = '>= 1.9.3'` but i am not sure we want to take the support out yet.